docs/archived/locked-inc-ftp/figures/firmware/source/tasks/sensor_task.c

```
@@ -146,10 +142,9 @@ static void internal_sensor_task_loop(void* const parameter)
 
 /* ========== Public API Implementation ========== */
 
-esp_err_t sensor_task_start(star_bus_manager_t* const     bus_manager,
-                            star_error_interface_t* const error_iface,
-                            const char* const             bus_name)
-{
+esp_err_t sensor_task_start(star_bus_manager_t *const bus_manager,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'sensor_task_start' of similar type ('int *const') are easily swapped by mistake

(bugprone-easily-swappable-parameters)